src/main/java/com/softserveinc/ita/pageobjects/CategoriesPage.java

```diff
@@ -7,11 +7,15 @@
     private final String subCategorySelector = "(//a[contains(@class , 'tile-cats__heading')])[1]";
     private final String productSelector = "(//div[@class = 'goods-tile__inner'])[1]";
```

Comment:
same

Reply:
But testcase says that we need first category
<img width="467" alt="image" src="https://user-images.githubusercontent.com/72382320/142930408-cbfa60a9-fec5-47f1-8554-43ebbf73d0de.png">